### PR TITLE
build: switch babel to swc

### PR DESCRIPTION
**Pull request by escapedcat**
Replace babel with [swc](https://swc.rs/docs/usage/swc-loader).

> SWC is 20x faster than Babel on a single thread and 70x faster on four cores.

### CODE REVIEW

**Comment by bumi** [comment]
we want no babel! yay! 🎉 drinks on me when this is merged.

File: .swcrc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "tsx": true
+    },
+    "transform": {
+      "react": {
+        "runtime": "automatic"
+      }
+    }
+  },
+  "env": {
+    "targets": [
```
Comment (escapedcat) [inline .swcrc:14]: Currently there's an issue with [browserslist](https://github.com/browserslist/browserslist) support so we set it directly in here

File: jest.config.ts [hunk 2]
```diff
@@ -15,6 +16,9 @@ module.exports = {
             },
           },
         },
+        env: {
+          targets: "defaults",
```
Comment (escapedcat) [inline jest.config.ts:20]: Use default-targets for jest tests.

**Review by bumi — APPROVED** [review]
works on my machine! :shipit:

**Comment by escapedcat** [comment]
yolo!

**Pull request merged**